src/js/elements/subindicator_filter/dropdown.js

```diff
@@ -0,0 +1,181 @@
+import {Observable} from "../../utils";
+
+
+export class DropdownModel extends Observable {
+    static EVENTS = {
+        'update': 'DropdownModel.update',     // triggered when new items are added or removed
```

Comment:
I know I did it this way, but could you please remove the quotes from the keys, i.e. update: `'DropdownModel.update' `instead of `'update': 'DropdownModel.update'`

Reply:
done